platform/android/app/src/main/kotlin/coop/polypoly/polypod/endpoint/Endpoint.kt

```diff
@@ -37,6 +37,43 @@ class Endpoint(
         return endpointsJson[endpointId]
     }
 
+    suspend fun uploadToServer(
```

Comment:
Hmm, that's an interesting `method` with very generic name. It is meant to send `errorMsg` to any given endpointId? Maybe consider renaming this method to `uploadError`.

Reply:
`uploadReport`? `uploadErrorReport`?

Reply:
yeah naming is not the best here, I tried to follow your advice of moving it to the Endpoint instead of an ErrorUploader wrapper. 
`uploadError` could work 👍🏽

Reply:
Not always an error...

Reply:
it's meant to be for error reports though... unless it changes use in the future

Reply:
Not really errors now, or in the future. In most cases, it's simply telemetry, directories present and missing... It's always a report, might contain some errors.

Reply:
to my knowledge it's used for app crashes atm

Reply:
Of course it's used for that, when it crashes. But just check out what's the content of the report here

https://github.com/polypoly-eu/polyPod/blob/66e097880f15ea2e55cfde92a6a230d39b33d902/features/google/src/views/report/reportWrapper.jsx#L18-L37

It says which ministories could be computed, and many other things. Please also check it's simply called `Report`, not `Error` or `Errors`.